saleor/product/migrations/0128_add_product_types_and_attributes_perm.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 3.1 on 2020-10-06 12:48
+
+from django.core.management.sql import emit_post_migrate_signal
+from django.db import migrations
+
+
+def update_groups_with_manage_products_with_new_permission(apps, schema_editor):
+    emit_post_migrate_signal(2, False, "default")
```

Comment:
I must use it because permissions are created in a `post_migrate` signal and otherwise `manage_product_types_and_attributes` permission doesn't exist here ([Django issue](https://code.djangoproject.com/ticket/23422)). 

I was also thinking about moving this function to separate migration but I am not sure which solution is better. @patrys

Reply:
I think this is fine but would add this comment that you wrote in the PR in the code, also with the link to Django issue.